pennylane/_grad.py

```diff
@@ -181,3 +182,206 @@ def _jacobian_function(*args, **kwargs):
         return _np.stack([_jacobian(func, arg)(*args, **kwargs) for arg in argnum]).T
 
     return _jacobian_function
+
+
+def _fd_first_order_centered(f, argnum, delta, *args, idx=None, **kwargs):
```

Comment:
Please, see this [discussion](https://github.com/PennyLaneAI/adrs/pull/7#discussion_r589831821)  in the ADR about indicating the finite-difference type in the function names. We can discuss it here and go with the one you like the most.

Reply:
@ixfoduap since this is a private function (and not user facing), it is generally okay to be a little bit more explicit/verbose.

But if it were a public, user-facing function, I would agree!